release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -119,6 +119,46 @@ With this release, migration from the OpenShift SDN network plugin to OVN-Kubern
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
+{product-title} 4.16 introduces volume cloning for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator as a Technology Preview feature. Volume cloning duplicates an existing persistent volume to help protect against data loss in {product-title}. 
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator] and xref:..//storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
+
+[id="ocp-4-16-storage-node-expansion-secret"]
+==== Node Expansion Secret is generally available
+The Node Expansion Secret feature allows your cluster to expand storage volumes, even when access to those volumes requires a secret (for example, a credential for accessing a SAN fabric) to perform the node expand operation. {product-title} 4.16 supports this feature as generally available.
+
+[id="ocp-4-16-vsphere-max-snapshot"]
+==== Changing vSphere CSI maximum number of snapshots is generally available
```

Comment:
If I remember correctly, it is a TP feature, right?
In that case, we should update the TP table accordingly.

Reply:
Ignore me due to: 
https://redhat-internal.slack.com/archives/GK0DA0JR5/p1715754771284149

Reply:
@lpettyjo As Wei pointed out - GA is not yet certain for this feature. I'll keep you posted.

Reply:
The link to the content is there, but commented out to avoid a publishing error.